### PR TITLE
Mergify

### PR DESCRIPTION
**Pull request by cosydney**
- [x] Internal linking: How many internal links did you put in the article? 1
- [x] Should it be added in `our axolo.co/resources` page? (if yes, add it to the list)

CleanShot 2023-09-29 at 11 11 37@2x

### CODE REVIEW

**Comment by cosydney** [comment]
[Slack File: image.png](https://files.slack.com/files-pri/T016LMS8GQN-F05UDAKAUHK/image.png)

File: data/p/streamline-pull-request-with-merge-queue.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+Standardize Practices: Ensure your team is on the same page regarding code review standards, PR sizes, and communication protocols. The right Merge Queue can help enforce these practices.
+
+Automate: Deploy automation for code reviews, dependency checks, and merging wherever possible to minimize human error and speed up the PR lifecycle.
+
+Measure and Iterate: After implementation, monitor key metrics such as time-to-merge, review efficiency, and code quality. Use this data to refine your approach and make necessary adjustments.
+
+By making a Merge Queue work for you, you're not just optimizing a single aspect of software development but elevating your entire workflow.
+
+Give Mergify a Try and avoid pull request frustration. [Check out our docs.](https://docs.mergify.com/)
```
Comment (acoudouy) [inline data/p/streamline-pull-request-with-merge-queue.mdx:156]: bizarre cette majuscule sur try non?

**Review by acoudouy — APPROVED** [review]
lgtm! :)

File: data/p/streamline-pull-request-with-merge-queue.mdx (hunk 1 above)
Comment (cosydney) [reply data/p/streamline-pull-request-with-merge-queue.mdx:156]: chatgpt je pense

**Pull request merged**